From 36a1f07bc9cd9c617d1764f224eaafbfbb2a3ba4 Mon Sep 17 00:00:00 2001
From: hwg <hwg@rock-chips.com>
Date: Sun, 22 Mar 2015 10:34:43 +0800
Subject: [PATCH] Revert "Add flags parameter to get_country_code template"

This reverts commit a3d5a3f5affd45d9aa21daf04d4eb543fd3ed444.

Conflicts:
	include/linux/wlan_plat.h
---
 include/linux/wlan_plat.h | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/include/linux/wlan_plat.h b/include/linux/wlan_plat.h
index 8e8b06f1ba4a..fa961ef6f490 100644
--- a/include/linux/wlan_plat.h
+++ b/include/linux/wlan_plat.h
@@ -15,8 +15,6 @@
 #ifndef _LINUX_WLAN_PLAT_H_
 #define _LINUX_WLAN_PLAT_H_
 
-#define WLAN_PLAT_NODFS_FLAG	0x01
-
 struct wifi_platform_data {
 	int (*set_power)(int val);
 	int (*set_reset)(int val);
@@ -24,7 +22,7 @@ struct wifi_platform_data {
 	void *(*mem_prealloc)(int section, unsigned long size);
 	int (*get_mac_addr)(unsigned char *buf);
 	int (*get_wake_irq)(void);
-	void *(*get_country_code)(char *ccode, u32 flags);
+	void *(*get_country_code)(char *ccode);
 };
 
 #endif
-- 
2.35.3

